docs/import/add_disaster_startup.js

```diff
@@ -0,0 +1,23 @@
+import {authenticateToFirebase, Authenticator} from '../authenticate.js';
+import SettablePromise from '../settable_promise.js';
+import {enableWhenReady, SENTINEL_NEW_DISASTER_VALUE, toggleState} from './add_disaster.js';
+import TaskAccumulator from './task_accumulator.js';
+
+export {taskAccumulator};
+
+// 2 tasks: EE authentication, page load
+const taskAccumulator = new TaskAccumulator(2, enableWhenReady);
+
+// Necessary for listAssets.
+ee.data.setCloudApiEnabled(true);
+
+// TODO: do something with this promise, pass it somewhere - probably once
```

Comment:
Isn't this promise a prerequisite for anything else with Firebase happening? Shouldn't enableWhenReady not run until this happens?

Reply:
Yes......tbh not sure what I was thinking here. Thanks.

Reply:
I don't think you'll need to pass it anywhere, so you could just scrap the SettablePromise and make the token receiver:

(token) => authenticateToFirebase(token).then(() => taskAccumulator.taskCompleted())